res/i18n/fr/labels.json

```diff
@@ -543,7 +543,7 @@
 		"num": "#"	
 	},
 	"spectrogram": "Spectrogramme",
-	"spread": "Propagation",
+	"spread": "Diffusion",
```

Comment:
It could be "étalement" too. In fact, spread in the music contexte is often translated to "étalement".

@sadko4u : could you please tell us about what it does/is? If "spread" is thought as "spreading some strawberry jam on bread", then "étalement" is the way to go. If not, your explanation will help us to find the best translation match.

Reply:
From the trigger documentation: 'Dynamics spread - this knobs allows to add some dynamics to output signal.'
So here 'Spread' means that there is some amount of randomness is added to the velocity of the output signal.

Reply:
Thank @sadko4u. I'm all good for either Propagation, Diffusion, or Étalement then.
My preference would be "étalement.
@wargreen: ping